src/components/PlaySongs/PlaySongs.tsx

```diff
@@ -104,49 +141,122 @@ const PlaySongs = () => {
         setCanChangeSong(true);
     };
 
-    const mute = () => {
-        player.mute();
-        setIsMuted(true);
+    const toggleMute = () => {
+        if (!isMuted) {
+            player.mute()
+          }
+          if (isMuted) {
+            player.unMute()
+          }
+          setIsMuted(!isMuted)
     };
 
-    const unmute = () => {
-        player.unMute();
-        setIsMuted(false);
-    };
+    const closePlayer = useCallback(() => {
+        console.log("no more songs")
+        setPartyJoined(false)
+        setPlaybackStarted(false)
+        setPlayedSongs([])
+        setCanChangeSong(true)
+        setIsMuted(false)
+        setIsPlaying(false)
+        setPlayer(null)
+        if (owner && partyJoined) {
+            endParty();
+        }
+    }, [endParty, owner, partyJoined])
 
     useEffect(() => {
-        if (!songs) {
-            return;
+        if (currentUser.id === currentPlaylist.owner) {
```

Comment:
instead of this state and useffect, you can just create a variable:
const isOwner = currentUser.id === currentPlaylist.owner

Reply:
fixed